client.py

```diff
@@ -5,57 +5,60 @@
 from discord.ext import commands
 
 import data
-from cogs import *
+
 
 class TTSBot(commands.Bot):
     def __init__(self):
-        super().__init__(
-            command_prefix='$',
-            intents=discord.Intents.default())
+        super().__init__(command_prefix="$", intents=discord.Intents.default())
         self.data = data.Data()
         self.set_env_vars()
 
     def set_env_vars(self):
         self.TOKEN, self.GUILD_NAME = self.data.get_env_vars()
-        if self.TOKEN == None or self.GUILD_NAME == None:
-            print("ERROR in .env file, double check format in top-level README.md")
+        if not all(self.TOKEN or self.GUILD_NAME):
```

Comment:
Oops, this needs to be something along the lines of...
```suggestion
        if not all([self.TOKEN, self.GUILD_NAME]):
```
actually